src/Sylius/Component/User/Model/User.php

```diff
@@ -35,13 +35,6 @@ class User implements UserInterface, \Stringable
      */
     protected $usernameCanonical;
 
-    /**
-     * Random data that is used as an additional input to a function that hashes a password.
-     *
-     * @var string|null
-     */
-    protected $salt;
```

Comment:
`getSalt()` probably needs to be removed as well.

![image](https://github.com/Sylius/Sylius/assets/870747/31599a18-6852-42ca-ac4c-9015cfbc8ec1)

Reply:
![image](https://github.com/Sylius/Sylius/assets/870747/150d6210-de35-4e01-84d9-26f720c77837)

Reply:
fixed